lib/ESPEasySerial/ESPeasySerial.h

```diff
@@ -41,19 +41,32 @@
 # include <ESPEasySoftwareSerial.h>
 #endif // ifndef DISABLE_SOFTWARE_SERIAL
 
+#ifndef HAS_SERIAL2
+#if defined(ESP32S2) || defined(ESP32C3) || defined(ESP8266)
+  #define HAS_SERIAL2 0
+# elif defined(ESP32S3)
+  #define HAS_SERIAL2 1
+# elif defined(ESP32_CLASSIC)
+  #define HAS_SERIAL2 1
+# else
+  static_assert(false, "Implement processor architecture");
```

Comment:
You might want to hint at ESPEasySerial in the message, or should the filename be explicit enough?

Reply:
Not sure what you mean here.

Reply:
The message doesn't include 'ESPEasySerial', but the assertion will probably include the filename, so I'm not sure an explict mention is needed.

Reply:
Already worked around it by defining this in only 1 place using the defines of the ESP32-xx HAL defines :)